ledger/participant-integration-api/src/main/scala/platform/store/backend/common/CommonStorageBackend.scala

```diff
@@ -24,43 +24,50 @@ import com.daml.platform.store.Conversions.{
 }
 import com.daml.lf.data.Ref
 import com.daml.lf.data.Ref.PackageId
+import com.daml.logging.{ContextualizedLogger, LoggingContext}
+import com.daml.platform.common.MismatchException
 import com.daml.platform.store.Conversions
 import com.daml.platform.store.SimpleSqlAsVectorOf.SimpleSqlAsVectorOf
 import com.daml.platform.store.appendonlydao.JdbcLedgerDao.{acceptType, rejectType}
-import com.daml.platform.store.backend.StorageBackend
+import com.daml.platform.store.backend.{ParameterStorageBackend, StorageBackend}
 import com.daml.platform.store.backend.StorageBackend.RawTransactionEvent
 import com.daml.platform.store.entries.{ConfigurationEntry, PackageLedgerEntry, PartyLedgerEntry}
 import com.daml.scalautil.Statement.discard
-
-import scala.collection.mutable
+import scalaz.syntax.tag._
 
 private[backend] trait CommonStorageBackend[DB_BATCH] extends StorageBackend[DB_BATCH] {
 
+  private val logger: ContextualizedLogger = ContextualizedLogger.get(this.getClass)
+
   // Ingestion
 
-  private val preparedDeleteIngestionOverspillEntries: List[String] =
+  private val SQL_DELETE_OVERSPILL_ENTRIES: List[SqlQuery] =
```

Comment:
constant naming is camelcase starting with capital in scala

Reply:
IMO, consistency within a project is more important than following naming standards. Almost everywhere in our code, SQL constants are all uppercase.

Reply:
well, strictly speaking this is not an SQL constant in that sense, but a list of SQL constants ;)
but you are right we really seem to have this pattern established.
after we cleaned dao/ most of it will disappear, and the rest should be rewritten to string interpolation anyway, so we could part from this approach if we want then. (I created a cleanup task DPP-553 to capture this aspect)